custom/src/main/java/com/splunk/opentelemetry/resource/ResourceAttributesToSystemProperties.java

```diff
@@ -42,43 +36,12 @@ public void beforeByteBuddyAgent() {}
 
   @Override
   public void afterByteBuddyAgent() {
-    Attributes attributes = getResource().getAttributes();
+    Attributes attributes = Resource.getDefault().getAttributes();
```

Comment:
FYI, there's a chance this will change again, pending the resolution of https://github.com/open-telemetry/opentelemetry-java/issues/2796

Reply:
I know, I'm observing this issue - but thanks for letting me know.